dataactbroker/handlers/accountHandler.py

```diff
@@ -660,52 +660,54 @@ def listUsersWithStatus(self):
+            status = self.interfaces.jobDb.getSubmissionStatus(submission)
             if submission.user_id is None:
                 submission_user_name = "No user"
             else:
                 submission_user_name = self.interfaces.userDb.getUserByUID(submission.user_id).name
-            submissionDetails.append({"submission_id": submission.submission_id, "last_modified": submission.updated_at.strftime('%m/%d/%Y'),
-                                      "size": total_size, "status": status, "errors": error_count, "reporting_start_date": str(submission.reporting_start_date),
-                                      "reporting_end_date": str(submission.reporting_end_date), "user": {"user_id": submission.user_id,
-                                                                                                    "name": submission_user_name}})
-        return JsonResponse.create(StatusCode.OK, {"submissions": submissionDetails})
-
-    def listSubmissionsByCurrentUser(self):
-        """ List all submission IDs associated with the current user ID """
-        user_id = LoginSession.getName(flaskSession)
-        user = self.interfaces.userDb.getUserByUID(user_id)
-        submissions = self.interfaces.jobDb.getSubmissionsByUserId(user_id)
-        submissionDetails = []
-        for submission in submissions:
-            job_ids = self.interfaces.jobDb.getJobsBySubmission(submission.submission_id)
-            total_size = 0
-            for job_id in job_ids:
-                file_size = self.interfaces.jobDb.getFileSize(job_id)
-                total_size += file_size if file_size is not None else 0
-
-            status = self.interfaces.jobDb.getSubmissionStatus(submission.submission_id)
-            error_count = sumNumberOfErrorsForJobList(submission.submission_id)
-            submissionDetails.append(
-                {"submission_id": submission.submission_id, "last_modified": submission.updated_at.strftime('%m/%d/%Y'),
-                 "size": total_size, "status": status, "errors": error_count, "reporting_start_date": str(submission.reporting_start_date),
-                                      "reporting_end_date": str(submission.reporting_end_date), "user": {"user_id": str(user_id),
-                                                                                                    "name": user.name}})
-        return JsonResponse.create(StatusCode.OK, {"submissions": submissionDetails})
+            submission_details.append({"submission_id": submission.submission_id,
+                                       "last_modified": submission.updated_at.strftime('%m/%d/%Y'),
+                                       "size": total_size, "status": status, "errors": submission.number_of_errors,
+                                       "reporting_start_date": str(submission.reporting_start_date),
+                                       "reporting_end_date": str(submission.reporting_end_date),
```

Comment:
Outside the scope for this PR, but do you know we're using two different date formats here?